modules/tensor_mechanics/src/materials/ComputeRotatedElasticityTensorBase.C

```diff
@@ -27,7 +29,8 @@ ComputeRotatedElasticityTensorBaseTempl<is_ad>::ComputeRotatedElasticityTensorBa
   : ComputeElasticityTensorBaseTempl<is_ad>(parameters),
     _Euler_angles(this->template getParam<Real>("euler_angle_1"),
                   this->template getParam<Real>("euler_angle_2"),
-                  this->template getParam<Real>("euler_angle_3"))
+                  this->template getParam<Real>("euler_angle_3")),
+    _rotation_matrix(this->template getParam<RealTensorValue>("rotation_matrix"))
```

Comment:
Conditional on isParamValid. Assign identity matrix otherwise

Reply:
@dschwen Does `RealTensorValue` have some kind of "identity" method? I'm not sure where that type is defined.

Reply:
OK I just typed in the identity matrix manually in the conditional. Also had to use `this->template getParam<RealTensorValue>()`.

Reply:
A rankTwoTensor has an identity constructor.

Reply:
Sure, but does `RankTwoTensor == RealTensorValue`? As far as I can tell, `RealTensorValue` is a templated type of `TensorValue`, but then I get lost.

Reply:
So is RankTwoTensor, since https://github.com/idaholab/moose/pull/11403

Reply:
But that doesn't necessarily mean that RealTensorValue has the same, right? If so I can't seem to find it (or at least any examples of it). For now I just built it manually in https://github.com/jessecarterMOOSE/moose/commit/3cf73eabad4ef2726a480e686c1a2b5e5ef3b95b.